src/client/Microsoft.Identity.Client/Region/RegionSource.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.;
+
+namespace Microsoft.Identity.Client.Region
```

Comment:
nit: could this be added to the constants file instead? w/a comment under Region?